uaclient/tests/test_cli.py

```diff
@@ -250,6 +253,36 @@ def test_user_facing_error_handled_gracefully(
         assert msg in error_log
         assert "Traceback (most recent call last):" in error_log
 
+    @pytest.mark.parametrize("caplog_text", [logging.ERROR], indirect=True)
```

Comment:
Do we not capture error logging by default?

Reply:
oops we do. it would only make sense to do this if we were limiting to WARNING or DEBUG messages and exact matching I guess.